adam/runtime/wrapper.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+HADOOP_PREFIX=/opt/apache-hadoop
+HADOOP_CONF_DIR=${HADOOP_PREFIX}/etc/hadoop
+ln -s ${HADOOP_PREFIX} /usr/local/hadoop
+
+# overwrite hostname in conf
+sed "s/HOSTNAME/${YARN_MASTER:-localhost}/g" $HADOOP_PREFIX/etc/hadoop/core-site.xml.template > $HADOOP_PREFIX/etc/hadoop/core-site.xml
```

Comment:
This doesn't get used (since we're no longer setting up YARN), but YARN_MASTER --> localhost if unset.

I'll remove this.